eth/tools/fixtures/fillers/_utils.py

```diff
@@ -72,7 +73,7 @@ def calc_state_root(state: AccountState, account_db_class: Type[BaseAccountDB])
 
 def generate_random_keypair() -> Tuple[bytes, Address]:
     key_object = keys.PrivateKey(pad32(int_to_big_endian(random.getrandbits(8 * 32))))
-    return key_object.to_bytes(), key_object.public_key.to_canonical_address()
+    return key_object.to_bytes(), cast(Address, key_object.public_key.to_canonical_address())
```

Comment:
I believe this could get rid of the `cast` and simply be `Address(key_object.public_key.to_canonical_address())`

Reply:
@cburgdorf I didn't know we could do that! Thankyou for pointing it out, will change it immediately.